L1Trigger/TrackFindingTracklet/src/TrackletEventProcessor.cc

```diff
@@ -507,34 +502,53 @@ void TrackletEventProcessor::printSummary() {
-      << "Cleaning              " << setw(10) << cleanTimer_.ntimes() << setw(20) << setprecision(3)
-      << cleanTimer_.avgtime() * 1000.0 << setw(20) << setprecision(3) << cleanTimer_.tottime() << "\n"
-      << "Add Stubs             " << setw(10) << addStubTimer_.ntimes() << setw(20) << setprecision(3)
-      << addStubTimer_.avgtime() * 1000.0 << setw(20) << setprecision(3) << addStubTimer_.tottime() << "\n"
-      << "VMRouter              " << setw(10) << VMRouterTimer_.ntimes() << setw(20) << setprecision(3)
-      << VMRouterTimer_.avgtime() * 1000.0 << setw(20) << setprecision(3) << VMRouterTimer_.tottime() << "\n"
-      << "TrackletEngine        " << setw(10) << TETimer_.ntimes() << setw(20) << setprecision(3)
-      << TETimer_.avgtime() * 1000.0 << setw(20) << setprecision(3) << TETimer_.tottime() << "\n"
-      << "TrackletEngineDisplaced" << setw(10) << TEDTimer_.ntimes() << setw(20) << setprecision(3)
-      << TEDTimer_.avgtime() * 1000.0 << setw(20) << setprecision(3) << TEDTimer_.tottime() << "\n"
-      << "TripletEngine         " << setw(10) << TRETimer_.ntimes() << setw(20) << setprecision(3)
-      << TRETimer_.avgtime() * 1000.0 << setw(20) << setprecision(3) << TRETimer_.tottime() << "\n"
-      << "TrackletCalculator    " << setw(10) << TCTimer_.ntimes() << setw(20) << setprecision(3)
-      << TCTimer_.avgtime() * 1000.0 << setw(20) << setprecision(3) << TCTimer_.tottime() << "\n"
-      << "TrackletCalculatorDisplaced" << setw(10) << TCDTimer_.ntimes() << setw(20) << setprecision(3)
-      << TCDTimer_.avgtime() * 1000.0 << setw(20) << setprecision(3) << TCDTimer_.tottime() << "\n"
-      << "ProjectionRouter      " << setw(10) << PRTimer_.ntimes() << setw(20) << setprecision(3)
-      << PRTimer_.avgtime() * 1000.0 << setw(20) << setprecision(3) << PRTimer_.tottime() << "\n"
-      << "MatchEngine           " << setw(10) << METimer_.ntimes() << setw(20) << setprecision(3)
-      << METimer_.avgtime() * 1000.0 << setw(20) << setprecision(3) << METimer_.tottime() << "\n"
-      << "MatchCalculator       " << setw(10) << MCTimer_.ntimes() << setw(20) << setprecision(3)
-      << MCTimer_.avgtime() * 1000.0 << setw(20) << setprecision(3) << MCTimer_.tottime() << "\n"
-      << "MatchProcessor        " << setw(10) << MPTimer_.ntimes() << setw(20) << setprecision(3)
-      << MPTimer_.avgtime() * 1000.0 << setw(20) << setprecision(3) << MPTimer_.tottime() << "\n"
-      << "FitTrack              " << setw(10) << FTTimer_.ntimes() << setw(20) << setprecision(3)
-      << FTTimer_.avgtime() * 1000.0 << setw(20) << setprecision(3) << FTTimer_.tottime() << "\n"
-      << "PurgeDuplicate        " << setw(10) << PDTimer_.ntimes() << setw(20) << setprecision(3)
-      << PDTimer_.avgtime() * 1000.0 << setw(20) << setprecision(3) << PDTimer_.tottime();
+  edm::LogVerbatim("Tracklet") << "Process             Times called   Average time (ms)      Total time (s)";
```

Comment:
the previous usage is preferable: issuing one long message is better than issuing a bunch of separate messages (fewer MSG headers etc.)

Reply:
cleaned this up to use single longer messages instead